styles.css

```diff
@@ -39,17 +36,19 @@
                 align-items: end;
                 padding: 13px 0px 13px 0px;
                 gap: 39px;
+                flex-wrap: wrap;
             }
 
             .header{
                 display: flex;
                 color: #fff;
                 flex-direction: row;
                 align-items: center;
-                padding: 23px 164px 23px 136px;
+                padding: 23px 0px 23px 0px;
```

Comment:
you can make it even shorter: `23px 0` is same as `23px 0 23px 0`

Reply:
done)